.github/workflows/pr.yaml

```diff
@@ -1,17 +1,17 @@
-# This sets labels and reviewers.
-name: Labels & reviewers.
-
+name: "Pull Request Labeler"
 on:
-  - pull_request
+  pull_request_target:
```

Comment:
The big change is on here. The event allows the GITHUB_TOKEN to contain all the permissions required to labels the PR, see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target for more details.